File: examples-sdk/java/101c-vilapp-multi-pipeline-benchmark/Main.java
// 101c-vilapp-multi-pipeline-benchmark — Java SDK equivalent
// Compile: vil compile --from java --input 101c-vilapp-multi-pipeline-benchmark/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("multi-pipeline-bench", 3090);
        ServiceProcess pipeline = new ServiceProcess("pipeline");
        server.service(pipeline);
        server.compile();
    }
}

File: examples-sdk/kotlin/101c-vilapp-multi-pipeline-benchmark/main.kt
// 101c-vilapp-multi-pipeline-benchmark — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 101c-vilapp-multi-pipeline-benchmark/main.kt --release

fun main() {
    val server = VilServer("multi-pipeline-bench", 3090)
    val pipeline = ServiceProcess("pipeline")
    server.service(pipeline)
    server.compile()
}
